torchvision/prototype/transforms/_auto_augment.py

```diff
@@ -1,16 +1,36 @@
+import dataclasses
 import math
-from typing import Any, Dict, Tuple, Optional, Callable, List, cast, TypeVar
+from typing import Any, Dict, Tuple, Optional, Callable, List, cast, Iterator
 
 import PIL.Image
 import torch
 from torchvision.prototype import features
-from torchvision.prototype.transforms import Transform, InterpolationMode, AutoAugmentPolicy, functional as F
+from torchvision.prototype.transforms import Transform, InterpolationMode, AutoAugmentPolicy, kernels as K
 from torchvision.prototype.utils._internal import apply_recursively
+from torchvision.transforms import functional as _F
 
-from ._utils import query_image
+from ._utils import query_image, get_image_size, get_image_num_channels
 
-K = TypeVar("K")
-V = TypeVar("V")
+
+@dataclasses.dataclass
```

Comment:
It seems like this version of AutoAugment is not the one we have on main. Many of the previously requested changes are undone here. I'm not going to further review this file and chat offline for its status.

Reply:
We will also need to include AugMix: #5411